package.json

```diff
@@ -45,6 +46,10 @@
     "predeploy": "yarn --ignore-optional",
     "deploy": "gulp docs:deploy"
   },
+  "resolutions": {
```

Comment:
Have just read up on this property, very interesting. Haven't seen it before 💠

Reply:
Got a link?

Reply:
https://yarnpkg.com/lang/en/docs/selective-version-resolutions/

Reply:
Yeh, it's super useful – I found it became an issue when I had updated to use the new colours in Fozzie, but it was getting stuck on the older version due to the includes in the header and footer, throwing Sass errors around variables that didn't exist.

This let's you override the depended version if you know that it won't break anything else by doing so :)